specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/CommonDefinitions.json

```diff
@@ -50,6 +50,18 @@
           "description": "Container image tag.",
           "type": "string"
         },
+        "imageType": {
+          "description": "The type of the image. Set to CloudBuild to let the system manages the image, where user will not be able to update image through image field. Set to ContainerImage for user provided image.",
```

Comment:
>CloudBuild [](http://example.com/codeflow?start=56&length=10)

Is there a default if its not specified? Or should it be marked a required property?

Reply:
user can leave this one blank. RP will default to `ContainerImage` if nothing is provided